include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -2191,6 +2261,26 @@ MGTwoLevelTransfer<dim, LinearAlgebra::distributed::Vector<Number>>::
   AlignedVector<VectorizedArrayType> evaluation_data_fine;
   AlignedVector<VectorizedArrayType> evaluation_data_coarse;
 
+  // a helper function similar to AffineConstraints::distribute_local_to_global
+  // but working with local indices
+  const auto distribute_local_to_global =
+    [&](const auto &index, const auto &value, auto &global_vector) {
+      if ((distribute_local_to_global_ptr[index + 1] -
+           distribute_local_to_global_ptr[index]) == 0)
+        global_vector.local_element(index) += value;
+      else if (((distribute_local_to_global_ptr[index + 1] -
+                 distribute_local_to_global_ptr[index]) == 1 &&
+                distribute_local_to_global_entries
+                    [distribute_local_to_global_ptr[index]]
+                      .first == numbers::invalid_unsigned_int) == false)
```

Comment:
Can you explain what the `numbers::invalid_unsigned_int` case indicates? I kind of thought I knew it in the code above, but now I'm lost.

Reply:
I am marking this way homogeneous constraints. I have added some comments.

Reply:
So in case we would ever support inhomogeneous constraints here, we would use the value entry for the inhomogeneity, right?

Reply:
Yes.